src/central-server/admin-service/application/src/main/resources/permissions.yml

```diff
@@ -54,22 +54,22 @@
 - APPROVE_OWNER_CHANGE_REQUEST: [ XROAD_REGISTRATION_OFFICER ]
 - REVOKE_OWNER_CHANGE_REQUEST: [ XROAD_REGISTRATION_OFFICER ]
 ##
-- VIEW_MEMBER_CLASSES: [ XROAD_REGISTRATION_OFFICER, XROAD_SYSTEM_ADMINISTRATOR ]
-- ADD_MEMBER_CLASS: [ XROAD_SYSTEM_ADMINISTRATOR ]
-- EDIT_MEMBER_CLASS: [ XROAD_SYSTEM_ADMINISTRATOR ]
-- DELETE_MEMBER_CLASS: [ XROAD_SYSTEM_ADMINISTRATOR ]
+- VIEW_MEMBER_CLASSES: [ XROAD_SECURITY_OFFICER, XROAD_SYSTEM_ADMINISTRATOR ]
```

Comment:
According to [spec](https://github.com/nordic-institute/X-Road/blob/release-7.4/doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md#41-managing-the-member-classes), CRUD operations regarding member classes should be allowed to XROAD_SECURITY_OFFICER only.